### ISSUE TITLE
branch v2.1.3: not concatenating multiple images in slideshow

### ISSUE THREAD

**Issue by torvista** [mc12345678/zen-colorbox#21]
Hi,
for a product with main and additional images.

Dropped branch master into a new ZC156a install: click on any image and the lightbox correctly shows the 1 of 4 for example and onclick moves to the next image.

Dropped branch v2.1.3 into a new ZC156a install: click on any image and the lightbox shows one image only, same for each image.

**Comment by mc12345678** [comment]
I just did the same on a ZC 1.5.6aish site (meaning there are or may be some corrections captured in the v156 branch that are applied but not in 1.5.6a) for the commit just before attempting to change out the parameter rel= with data-colorbox= and went to the demo product `index.php?main_page=product_info&cPath=23&products_id=48`, selected any of the four images, received a popup with the number associated with the image's position and was able to select each of the prev/next options and cycle through the options.
Once I applied the "rel" change, then I received what you saw.

I have spent the evening investigating so that could address #20 and this issue.  Figured it out... Apparently the reference to the object was never really correctly made.  Yes "rel" is still used, but as part of data-cbox-rel... I have "updated" (force pushed) the change to using data-cbox-rel instead of just rel to the 2.1.3 branch.

I did also discover that `includes/templates/YOUR_TEMPLATE/jscript/jquery.colorbox-min.js` has been using a deprecated function that has become removed as of jQuery 3.x that was throwing a console error.  That solution has also been pushed to maintain operation.
